src/openrct2-ui/windows/EditorObjectSelection.cpp

```diff
@@ -1079,35 +1077,36 @@ static void window_editor_object_selection_paint(rct_window* w, rct_drawpixelinf
         ft.Add<rct_string_id>(STR_STRING);
         ft.Add<const char*>(description.c_str());
 
-        x = w->windowPos.x + w->widgets[WIDX_LIST].right + 4;
-        y += 15;
-        width = w->windowPos.x + w->width - x - 4;
+        screenPos = { w->windowPos.x + w->widgets[WIDX_LIST].right + 4, screenPos.y + 15 };
+        width = w->windowPos.x + w->width - screenPos.x - 4;
 
-        gfx_draw_string_left_wrapped(dpi, gCommonFormatArgs, x, y + 5, width, STR_WINDOW_COLOUR_2_STRINGID, COLOUR_BLACK);
+        gfx_draw_string_left_wrapped(
+            dpi, gCommonFormatArgs, screenPos.x, screenPos.y + 5, width, STR_WINDOW_COLOUR_2_STRINGID, COLOUR_BLACK);
     }
 
-    y = w->windowPos.y + w->height - (12 * 4);
+    screenPos.y = w->windowPos.y + w->height - (12 * 4);
```

Comment:
if you set screenPos.x to the value of `w->windowPos.x + w->width - 5` then that would simplify this considerably.